pipelines/smartseq2_by_plate/AggregateMetrics.py

```diff
@@ -0,0 +1,99 @@
+from crimson import picard
+import json
+import argparse
+import os
+import pandas as pd
+
+
+def AggregatePicardMetricsRow(filenames, output_name):
```

Comment:
This file feels like it belongs in the sctools repo. Also, can all of this be done using the Picard jar, or are there limitations that requires us to write our own code for this?